src/main.rs

```diff
@@ -1,12 +1,60 @@
 use std::{env, fs};
 
 
+
+fn find_file(path: String, file: String) {
+    let mut queue = Vec::new();
```

Comment:
Do you really need to use Vec here? Try to avoid and use your own custom structs